packages/medusa/src/migrations/1660040729000-payment_session_uniq_cart_id_providerId.ts

```diff
@@ -0,0 +1,21 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
```

Comment:
todo: my thought was to only have a single migration for this PR; so we should combine this migration with the one above :)

Reply:
My bad. Have merged the two 👍